deepspeed/module_inject/replace_module.py

```diff
@@ -0,0 +1,191 @@
+import copy
+import torch
+import deepspeed
+
+
+def replace_transformer_layer(orig_layer_impl,
```

Comment:
can we add a test case for this feature, to avoid being broken by future changes?

Reply:
This means have a test-case that initially uses the bert-modeling and the inject the kernel using this function. Can you please write a test-bench to do this? I have already tested this with BingBertSquad and see similar results between baseline and when using the injection-code.
Thanks Elton.